nautobot/docs/development/core/kubernetes-job-support.md

```diff
@@ -0,0 +1,509 @@
+# Kubernetes Job Support
+
+Kubernetes Job Support is added in Nautobot v2.4.0. This documentation is a end-to-end development guide on topics from how to set up your local kubernetes cluster with [minikube](https://minikube.sigs.k8s.io/docs/) and how to run a nautobot job in a remote kubernetes job pod.
+
+## Preliminary Setup
```

Comment:
It would be great to have `invoke` command to run all checks from this step and `Check Required Files` like permissions, kubectl installed, presence of required files. But it's more like feature request for future or some hackaton :)